lib/modules/dosomething/dosomething_kudos/dosomething_kudos.module

```diff
@@ -131,67 +126,120 @@ function dosomething_kudos_sort_by_taxonomy_term($a, $b) {
   return $result;
 }
 
+/**
+ * Get current user from supplied data array or global user.
+ *
+ * @param  array  $data
+ * @return null|string
+ */
+function dosomething_kudos_get_current_user($data = []) {
```

Comment:
Yeah, if I remove it as a URL param that gets passed in and along to numerous functions (I'm passing an `options` array around that had to contain the user id that could be passed via `/reportback-items?current_user=1219129`), then I can just do away with all this to begin with and rely on solely the `global $user` which is way simpler :)